spec/date.html

```diff
@@ -449,23 +455,20 @@ <h1>DateFromString ( _date_ )</h1>
     <emu-clause id="sec-temporal-todate" aoid="ToDate">
       <h1>ToDate ( _date_ )</h1>
       <emu-alg>
-        1. If Type(_date_) is String, then
-          1. Return ? DateFromString(_date_).
         1. If Type(_date_) is Object, then
           1. Let _result_ be the Record {
-            [[Year]]: *undefined*,
-            [[Month]]: *undefined*,
-            [[Day]]: *undefined*,
+            [[Year]]: 0,
+            [[Month]]: 1
+            [[Day]]: 1
             }.
```

Comment:
Consider whether it would be clearer to put the default values in the table, and if we could use something like "a new Record with all the internal slots given in the Internal Slot column in #table".